packages/ag-charts-community/src/chart/series/cartesian/areaSeries.ts

```diff
@@ -98,6 +105,69 @@ interface AreaSeriesStackContext {
     strokeSpans: LinePathSpan[];
 }
 
+/**
+ * Context object caching expensive lookups for createNodeData().
+ * Created once per createNodeData() call and passed to helper methods.
+ */
+interface AreaSeriesCreateNodeDatumContext {
```

Comment:
Should these interfaces share a base interface? There seems to be some crossover between various series.